### PR TITLE
Rfc6265bis tests

### PR DESCRIPTION
**Pull request by mozmark**
These are a port of @mikewest 's tests on http://rfc6265.biz/tests/ to WPT with @gsnedders ' Add alternate_hosts work (merged in #10166). I have more tests to add but I wanted to make sure the straight porting of these was OK first.

### CODE REVIEW

**Comment by annevk** [comment]
Yay! Please put these in `cookies/`; no need for a new top-level directory. Also, FWIW, the main reason this fails is due to linting errors: https://travis-ci.org/w3c/web-platform-tests/jobs/374399296#L538.

File: cookies/rfc6265bis/index.html [hunk 1]
```diff
@@ -0,0 +1,364 @@
+<!DOCTYPE html>
```
Comment (mikewest) [inline cookies/rfc6265bis/index.html]: We don't need this file. It made sense for the stand-alone site, it doesn't make sense as part of WPT.

File: cookies/rfc6265bis/prefix/__host.document-cookie.non-secure.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (mikewest) [inline cookies/rfc6265bis/prefix/__host.document-cookie.non-secure.html]: I'd suggest dropping the `rfc6265bis` directory. "prefix" is enough.

File: cookies/rfc6265bis/resources/testharness.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
+html {
```
Comment (mikewest) [inline cookies/rfc6265bis/resources/testharness.css]: We don't need this, or `testharness.js`, or `testharnessreport.js`.

File: cookies/rfc6265bis/prefix/__host.document-cookie.non-secure.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<script src="/cookies/rfc6265bis/resources/testharness.js"></script>
+<script src="/cookies/rfc6265bis/resources/testharnessreport.js"></script>
```
Comment (mikewest) [inline cookies/rfc6265bis/prefix/__host.document-cookie.non-secure.html]: These should both be using WPT's `testharness.js` and `testharnessreport.js`, here and in every other HTML file.

**Review by mikewest — COMMENTED** [review]
This looks like a good port, thanks for doing the work! A few comments inline.

**Review by mikewest — APPROVED** [review]
Thanks!

**Pull request merged**